requirements-dev.txt

```diff
@@ -1,8 +1,8 @@
 isort>=5.10.1,<6.0
-black>=22.10.0,<23.0
```

Comment:
Is this file written by you or a tool ?

If it is you, you can specify the version with the "Compatible release clause" : `~=22.10` (https://packaging.python.org/en/latest/specifications/version-specifiers/#id4)

If not, no choice.